DataBase001/Model/Route.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DataBase001.Model
+{
+    public class Route : Entity
+    {
+        public List<Guid> RoutOfSity { set; get; }
+        public Route(string name,List<Guid> RoutOfSity):base(name)
+        {
+            this.RoutOfSity = RoutOfSity;
+
```

Comment:
Называй нормально
```suggestion
        public List<Guid> CityIdentifiers { set; get; }
        public Route(string name,List<Guid> cityIdentifiers):base(name)
        {
            this.CityIdentifiers = cityIdentifiers;

```

* Нормально - не значит "в точности как я". Значит - понятно, и без грамматических ошибок (Rout и Sity не являются английскими словами)
* Подумай, действительно ли извне класса можно просто заменить список городов разовым присваиванием. Это нарушение принципа инкапсуляции.

Reply:
Как английский подтяну так буду нормально называть)